content/md/retailer-onboarder/first-steps/retailer-synchronization.md

```diff
@@ -16,21 +16,35 @@ You will find here all the necessary information related to synchronization:
 
 # Catalog structure
 
-During the supplier creation you will be able to link one or n families to a supplier.
+When you create a new supplier, you need to link families and catalog locales to a supplier.
 What does it mean and what is the impact?
 
-If you link the supplier "Abibas" to the "Shoes" and "Accessories" families, it means that he will supply you with the products belonging to these families, so you expect him to fill in product data for these 2 families.
+If you link the supplier "Abibas" to the "Shoes" and "Accessories" families, it means that they will supply you with the products belonging to these families in all the locales you selected, so you expect the supplier to fill in product data for these 2 families.
```

Comment:
```suggestion
If you link the supplier "Abibas" to the "Shoes" and "Accessories" families, it means that they will supply you with the products belonging to these families in all the locales you have selected, so you expect the supplier to fill in product data for these 2 families.
```

Reply:
it's a suggestion, I'm not totally sure ;) (but Google agreed)

Reply:
@camillebourjade already approved this sentence. Let's see what she thinks.

Reply:
If it was already reviewed by @camillebourjade  it's ok for me :)